nbs/01a_losses.ipynb

```diff
@@ -62,7 +62,15 @@
     "class BaseLoss():\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            axis:int=-1, #  Determines which axis on which self.decodes and self.activation occurs

</pre><p>Slight shortening <code>#  Axis self.decodes and self.activation occurs on</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/muellerzr/fastai-docment-sprint/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>